src/strava_gear/data.py

```diff
@@ -164,14 +166,16 @@ def all_rule_bike_ids(self) -> Set[BikeId]:
 @dataclass
 class Usage:
     distances: Dict[ComponentId, float] = field(default_factory=lambda: defaultdict(float))
+    elevation_gain: Dict[ComponentId, float] = field(default_factory=lambda: defaultdict(float))
```

Comment:
This one should probably be `elevation_gains` for consistency.

Reply:
Thank you, I've fixed it.